rts/lra/README.adoc

```diff
@@ -55,3 +55,44 @@ curl -X POST  "http://localhost:8081/trip/book?hotelName=Rex"
 curl -X POST  "http://localhost:8081/trip/book?flightNumber=BA123"
 ```
 
+## Running the demo on Minishift
+
+You will need to have installed Minishift and have the tools in your command line.
+
+We provisioned Minishift as:
+```bash
+minishift start --vm-driver=virtualbox
```

Comment:
the `--vm-driver` is not necesary, I expect, that's up to the user if he wants to run with kvm, virtualbox...